### PR TITLE
Fix parse go demo #964

### PR DESCRIPTION
**Pull request by richarda23**
fix for #964

### CODE REVIEW

**Review by lafita — APPROVED** [review]
Looks good, thanks! If the file is not too large, another option is to download it to the resources folder and parse it locally. We have been trying to avoid remote dependencies in BioJava, specially in tests and demos, although many still remain.

**Comment by richarda23** [comment]
it's 72kb, is that small enough ? Agree would be good to be local if possible

**Comment by lafita** [comment]
Yes, I think that counts as small! Thanks @richarda23

**Comment by richarda23** [comment]
- replaced url call with file
- logging doesn't output any results so replaced with system.out/err calls
- exit with error code if fails

File: biojava-ontology/src/main/java/demo/ParseGO.java [hunk 1]
```diff
@@ -27,42 +27,41 @@
...
+	 * @param args
+	 */
+	public static void main(String[] args) {
 
 		try {
-			URL url = new URL(u);
 
 			OboParser parser = new OboParser();
-			InputStream inStream = url.openStream();
+			InputStream inStream = OboParser.class.getResourceAsStream("/ontology/biosapiens.obo");
```
Comment (lafita) [inline biojava-ontology/src/main/java/demo/ParseGO.java:51]: Have you committed the file? I can't see it in the commits

**Review by josemduarte — APPROVED** [review]
LGTM thanks!

**Pull request merged**